cluster_tools/cluster_tools/schedulers/cluster_executor.py

```diff
@@ -492,8 +545,7 @@ def register_jobs(
                 with_preliminary_postfix(output_path), jobid_with_index
             )
 
-            fut.cluster_jobid = jobid
-            fut.cluster_jobindex = array_index
```

Comment:
I think this was there for debugging purposes? Probably won't hurt to keep it?

Reply:
Sure, I just got type errors for it and didn't find any usages. I'll re-add it, type-ignore it and add some comments.